src/schema.rs

```diff
@@ -236,219 +217,303 @@ fn renomination() -> TransitionSchema {
+                    },
+                    owned_rights: type_map! {
+                        OwnedRightType::Inflation => NoneOrMore,
+                        OwnedRightType::Assets => NoneOrMore,
+                        OwnedRightType::OpenEpoch => NoneOrOnce,
+                        OwnedRightType::BurnReplace => NoneOrMore,
+                        OwnedRightType::Renomination => NoneOrOnce
+                    },
+                    public_rights: none!()
+                }
+            },
+            field_types: type_map! {
+                // Rational: if we will use just 26 letters of English alphabet (and
+                // we are not limited by them), we will have 26^8 possible tickers,
+                // i.e. > 208 trillions, which is sufficient amount
+                FieldType::Ticker => TypeRef::ascii_string(),
+                FieldType::Name => TypeRef::ascii_string(),
+                // Contract text may contain URL, text or text representation of
+                // Ricardian contract, up to 64kb. If the contract doesn't fit, a
+                // double SHA256 hash and URL should be used instead, pointing to
+                // the full contract text, where hash must be represented by a
+                // hexadecimal string, optionally followed by `\n` and text URL
+                FieldType::Precision => TypeRef::u8(),
+                // We need this b/c allocated amounts are hidden behind Pedersen
+                // commitments
+                FieldType::IssuedSupply => TypeRef::u64(),
+                // Supply in either burn or burn-and-replace procedure
+                FieldType::BurnedSupply => TypeRef::u64(),
+                // While UNIX timestamps allow negative numbers; in context of RGB
+                // Schema, assets can't be issued in the past before RGB or Bitcoin
+                // even existed; so we prohibit all the dates before RGB release
+                // This timestamp is equal to 10/10/2020 @ 2:37pm (UTC)
+                FieldType::Timestamp => TypeRef::i64(),
+                FieldType::HistoryProof => TypeRef::bytes(),
+                FieldType::BurnUtxo => TypeRef::new("OutPoint")
+            },
+            owned_right_types: type_map! {
+                // How much issuer can issue tokens on this path. If there is no
+                // limit, than `core::u64::MAX` / sum(inflation_assignments)
+                // must be used, as this will be a de-facto limit to the
+                // issuance
+                OwnedRightType::Inflation => StateSchema::DiscreteFiniteField(DiscreteFiniteFieldFormat::Unsigned64bit),
+                OwnedRightType::Assets => StateSchema::DiscreteFiniteField(DiscreteFiniteFieldFormat::Unsigned64bit),
+                OwnedRightType::OpenEpoch => StateSchema::Declarative,
+                OwnedRightType::BurnReplace => StateSchema::Declarative,
+                OwnedRightType::Renomination => StateSchema::DataContainer
```

Comment:
I recall from last call that you said that `OwnedRightType::Renomination` should not be defined as a `DataContainer` but instead as `Declarative`